src/shared/components/userMessager/UserMessage.tsx

```diff
@@ -78,7 +78,8 @@ export default class UserMessager extends React.Component<
         if (
             !isWebdriver() &&
             !this.dismissed &&
-            (this.messageData.isComplete && this.shownMessage)
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)